docs/tutorial.md

```diff
@@ -394,7 +394,107 @@ What's going on here? Let's step through the code.
+attention to some easy-to-miss details for this approach.
+
+Let's start with a nonWF simulation similar to what we've seen previously.
+
+```{literalinclude} sample.slim
+```
+
+```{code-cell}
+%%bash
+slim -s 1 sample.slim
+```
+
+The first, most common method to extract individuals is simply to get all
+those that were alive at a particular time,
+using {meth}`SlimTreeSequence.individuals_alive_at`. For instance, to get
+the list of individual IDs of all those alive at the end of the
+simulation (i.e., zero time units ago), we could do:
+
+```{code-cell}
+orig_ts = pyslim.load("sample.trees")
+alive = orig_ts.individuals_alive_at(0)
+
+print(f"There are {len(alive)} individuals alive in the final generation.")
+```
+
+These are individual IDs. One way to take a sample of living individuals
+and write their SNPs to a VCF is:
+
+```{code-cell}
+np.random.seed(1)
+keep_indivs = np.random.choice(alive, 100, replace=False)
+ts = pyslim.SlimTreeSequence(msprime.mutate(orig_ts, rate=1e-8, random_seed=1))
+with open("example_snps.vcf", "w") as vcffile:
+    ts.write_vcf(vcffile, individuals=keep_indivs)
+```
+
+
+### Extracting individuals after simplification
+
+In cases where we want to obtain genotypes *after* simplification, we
+need to pay attention to details concerning individual nodes. As
+mentioned earlier, {meth}`.SlimTreeSequence.simplify` takes a list
+of nodes as input:
+
+```{code-cell}
+keep_nodes = []
+for i in keep_indivs:
+    keep_nodes.extend(orig_ts.individual(i).nodes)
+sts = rts.simplify(keep_nodes)
+```
+
+There are often some nodes retained
+after simplification that were not part of our sample. Furthermore, it
+is important to remember that the individual IDs get changed during
+simplification, so we cannot depend on our original list of individual
+IDs. Therefore, we will need to refer
+to the *node* IDs attached to our sample:
+
+```{code-cell}
+ts = pyslim.SlimTreeSequence(msprime.mutate(sts, rate=1e-8, random_seed=1))
+indivlist = []
+for i in ts.individuals_alive_at(0):
+    ind = ts.individual(i)
+    if ts.node(ind.nodes[0]).is_sample():                                                                              
+       indivlist.append(i)
+with open("example_snps.vcf", "w") as vcffile:
+    ts.write_vcf(vcffile, individuals=indivlist)
+```
+
+:::{note}
+The {meth}`.Node.is_sample` check is a necessary and easy to
+forget step. If this step is omitted, the program will attempt
+to extract genotypes for any isolated nodes, e.g. non-sample
+nodes that were part of the simplified tree.
+This will result in an error message.
+:::
+
+
+The above code uses {meth}`.Node.is_sample` to check whether a node
+was part of our sample. This information was saved during the
+simplification step, and is saved in the ``flags`` attribute of the
+node metadata. For example, below shows the first node
+from two individuals: one individual with a sample
+flag (Individual 0) and one non-sample individual (Individual 2).
```

Comment:
I'm not so sure about whether this goes here - the purpose is demonstrating how to interpret the `flags` attribute. So it should either say explicitly that `tskit.NODE_IS_SAMPLE` is `1`, and so if `node.flags & tskit.NODE_IS_SAMPLE` is nonzero then it's a sample... but this gets into explaining bitwise flags, and this might not be the right place for it.

Reply:
deleted this paragraph